### PR TITLE
Additional tiny processing editor / python console improvements

### PR DESCRIPTION
**Pull request by nirvn**
## Description
The main benefit is the shared editor color configuration between the python console and processing's script editor.

The "you are about to lose unsaved changes" dialog for the processing script editor was also improved to match the modeler (and QGIS generally speaking).

The rest is purely cosmetic.

## Checklist

> Reviewing is a process done by project maintainers, mostly on a volunteer basis. We try to keep the overhead as small as possible and appreciate if you help us to do so by completing the following items. Feel free to ask in a comment if you have troubles with any of them.

- [x] Commit messages are descriptive and explain the rationale for changes
- [x] Commits which fix bugs include `fixes #11111` in the commit message next to the description
- [x] Commits which add new features are tagged with `[FEATURE]` in the commit message
- [x] Commits which change the UI or existing user workflows are tagged with `[needs-docs]` in the commit message and contain sufficient information in the commit message to be documented
- [x] I have read the [QGIS Coding Standards](https://docs.qgis.org/testing/en/docs/developers_guide/codingstandards.html) and this PR complies with them
- [x] This PR passes all existing unit tests (test results will be reported by travis-ci after opening this PR)
- [x] New unit tests have been added for core changes
- [x] I have run [the `scripts/prepare-commit.sh` script](https://github.com/qgis/QGIS/blob/master/.github/CONTRIBUTE.md#contributing-to-qgis) before each commit

### CODE REVIEW

File: python/console/console.py [hunk 1]
```diff
@@ -193,7 +193,7 @@ def __init__(self, parent=None):
         self.saveFileButton = QAction(self)
         self.saveFileButton.setCheckable(False)
         self.saveFileButton.setEnabled(False)
-        self.saveFileButton.setIcon(QgsApplication.getThemeIcon("console/iconSaveConsole.svg"))
```
Comment (alexbruy) [inline python/console/console.py]: Should we drop this icon completely if it is not used anymore?

File: python/console/console.py [hunk 2]
```diff
@@ -243,7 +243,7 @@ def __init__(self, parent=None):
         self.runScriptEditorButton = QAction(self)
         self.runScriptEditorButton.setCheckable(False)
         self.runScriptEditorButton.setEnabled(True)
-        self.runScriptEditorButton.setIcon(QgsApplication.getThemeIcon("console/iconRunScriptConsole.svg"))
```
Comment (alexbruy) [inline python/console/console.py]: Should we drop this icon completely if it is not used anymore?

File: python/console/console_editor.py [hunk 3]
```diff
@@ -283,7 +283,9 @@ def get_end_pos(self):
     def contextMenuEvent(self, e):
         menu = QMenu(self)
         iconRun = QgsApplication.getThemeIcon("console/mIconRunConsole.svg")
-        iconRunScript = QgsApplication.getThemeIcon("console/iconRunScriptConsole.svg")
```
Comment (alexbruy) [inline python/console/console_editor.py]: Should we drop this icon completely if it is not used anymore?

**Review by alexbruy — APPROVED** [review]
Nice!

**Comment by nirvn** [comment]
@alexbruy , good catch, icons removed.

Here's how things look now using this PR:
screenshot from 2018-05-20 15-07-24

**Comment by nyalldawson** [comment]
Thanks for fixing that save dialog - I often get confused by the behaviour of that one!

**Pull request merged**